driver/Makefile

```diff
@@ -1,8 +1,25 @@
 MPIRUN_CALL ?= mpirun -n 6
 
 test:
-	pytest tests
+	pytest tests --ignore=tests/test_restart.py
```

Comment:
Is there any way to avoid this? e.g. can we have a `tests/main` and `tests/mpi` separation for the tests that need MPI?